### PR TITLE
Fix namespacing so it lines up properly with file names and paths

### PR DESCRIPTION
**Pull request by EraYaN**
**Changes**
This changes the namespace several classes are in to properly line up with their position in the file tree. There were some really weird ones in there. In total 33 namespaces changed. We shouldn't be declaring stuff in the System.Net namespace for example.

This might/will break plugins.

### CODE REVIEW

File: MediaBrowser.Providers/Manager/GenericPriorityQueue.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 using System.Collections.Generic;
 using System.Runtime.CompilerServices;
 
-namespace Priority_Queue
```
Comment (Bond-009) [inline MediaBrowser.Providers/Manager/GenericPriorityQueue.cs]: Leave this one, we should try and replace it.

File: Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@
 using MediaBrowser.Model.LiveTv;
 using Microsoft.Extensions.Logging;
 
-namespace Jellyfin.Server.Implementations.LiveTv.Listings
+namespace Emby.Server.Implementations.LiveTv.Listings
```
Comment (Bond-009) [inline Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs]: I'm not sure about this one.

Comment (JustAMan) [reply Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs]: I personally would rather see the `Emby.Server.Implementations` renamed to `Jellyfin.Server.Implementations` if that is not too much difficult...

File: MediaBrowser.Providers/Manager/GenericPriorityQueue.cs (hunk 1 above)
Comment (JustAMan) [reply MediaBrowser.Providers/Manager/GenericPriorityQueue.cs]: I side here with @Bond-009 - this just highlights it being an embedded 3rd party we want eventually to extract.
Maybe add a `FIXME` or `TODO` here and leave the namespace be like it was?

File: Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs (hunk 2 above)
Comment (Bond-009) [reply Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs]: We have to be careful if we do that, resources use the name of the csproj

**Comment by EraYaN** [comment]
Alright, renamed the `Priority_Queue` namespace.

**Pull request merged**